packages/cozy-client/src/CozyClient.js

```diff
@@ -663,6 +686,20 @@ client.query(Q('io.cozy.bills'))`)
     }
   }
 
+  /**
+   * Dehydrates and adds metadata before saving a document
+   *
+   * @param  {CozyClientDocument} doc - Document that will be saved
+   * @returns {CozyClientDocument}
```

Comment:
We should certainly create a new type here one day.

Reply:
OK.